html/xpath value finder (internal)/35629a5f-aad8-4c08-8946-7a46af449595.json

```diff
@@ -18,25 +18,30 @@
         }
       },
       "inputFunctions": {
-        "6c326055-5409-47ab-8197-305240922f8e": "const xpath = require(\"xpath\");\nconst parse5 = require(\"parse5\");\nconst xmlser = require(\"xmlserializer\");\nconst dom = require(\"xmldom\").DOMParser;\n\nconst { document, query } = data;\nconst doc = new dom({ \n  errorHandler: {\n    warning: () => {},\n    error: console.error\n  }\n}).parseFromString(document);\n\nconst select = xpath.useNamespaces({ \"x\": \"http://www.w3.org/1999/xhtml\" });\nconst nodes = select(query, doc);\n\nconst values = nodes.map((node) => node.toString());\n\noutputs.results(values, tag)"
+        "6c326055-5409-47ab-8197-305240922f8e": "const xpath = require(\"xpath\");\nconst parse5 = require(\"parse5\");\nconst xmlser = require(\"xmlserializer\");\nconst dom = require(\"xmldom\").DOMParser;\n\nconst { document, query } = data;\nconst doc = new dom({ \n  errorHandler: {\n    warning: () => {},\n    error: (e) => {outputs.error(e.toString(), tag)}\n  }\n}).parseFromString(document);\n\nconst select = xpath.useNamespaces({ \"x\": \"http://www.w3.org/1999/xhtml\" });\nconst nodes = select(query, doc);\n\nconst values = nodes.map((node) => node.toString());\n\noutputs.results(values, tag)"
```

Comment:
I think error output should be at least of type `{error: string}`

Reply:
Good catch!